diff -urN chromium-bsu-0.9.15/src/MenuGL.cpp chromium-bsu-0.9.15-patched/src/MenuGL.cpp
--- chromium-bsu-0.9.15/src/MenuGL.cpp	2011-02-14 13:54:03.000000000 +1000
+++ chromium-bsu-0.9.15-patched/src/MenuGL.cpp	2011-12-13 23:59:04.000000000 +1100
@@ -465,7 +465,7 @@
 		case SkillLevel: 
 			level = config->gameSkillBase();
 			tmp = (int)((level+0.05)*10.0);
-			sprintf(buf, skillString(tmp));
+			sprintf(buf, "%s", skillString(tmp));
 			break;
 		case Graphics: 
 			level = config->gfxLevel()/2.0;
